.github/workflows/cd.yml

```diff
@@ -189,11 +189,11 @@ jobs:
           git commit -m "build: replace sideloading placeholders"
 
       - name: disable chat participant environment variable
-        if: ${{ github.event_name == 'workflow_dispatch' && (github.event.inputs.preid != 'alpha') }}
+        if: ${{ github.event_name == 'workflow_dispatch' && (github.event.inputs.preid != 'alpha' && github.event.inputs.preid != 'beta') }}
```

Comment:
It seems that the script could be removed since we use function instead of const variable?

Reply:
If we need to hide them in the stable release TTK, we also need to check the preid.